tests/unit/cn/cn_tree_test.c

```diff
@@ -536,10 +528,10 @@ test_init(struct test *t, struct test_params *params, struct mtf_test_info *lcl_
     t->mtf = lcl_ti;
 }
 
-static u32
-num_kvsets_in_node(u32 node_level, u32 node_offset)
+static uint64_t
+num_kvsets_in_node(uint64_t nodeid)
 {
-    return 1 + node_offset % 3;
+    return 1 + (nodeid % 4);
```

Comment:
Ooops, was mod 3 there for any particular reason?  Seems to work ok with mod 4...

Reply:
any leaf node should work.  the mocked tree used to have 0,1,2 kvsets in cycle (or something like that).  and the mod 3 was to get a node with > 0 kvsets.  I changed to mock to have 3,4,5 in a cycle.  so it should no longer matter which leaf you get.